tests/unit/components/icon-extention-with-title/IconExtentionWithTitle.spec.jsx

```diff
@@ -22,7 +22,7 @@ describe('IconExtentionWithTitle test', () => {
   })
 
   it('Should render description', () => {
-    const description = screen.getByText('13 common.megabytes')
+    const description = screen.getByText('13 B')
```

Comment:
i think you have to find it not by text B but by translation and add translation for bytes, kilobytes, megabytes

Reply:
I don't think we need translations for these (it will always be B, KB and MB

Reply:
done